packages/compass-web/src/connection-storage.tsx

```diff
@@ -96,6 +110,40 @@ function getMetricsIdAndType(
   };
 }
 
+function getInstanceSize(
+  clusterDescription: ClusterDescription
+): string | undefined {
+  return getFirstInstanceSize(clusterDescription.replicationSpecList);
```

Comment:
I copied this kinda literally from MMS code, but I suppose I might as well roll getInstanceSize in here.